### ISSUE TITLE
error with rule "no-empty-declaration"

### ISSUE THREAD

**Issue by kellyselden** [onechiporenko/eslint-plugin-ember-cleanup#4]
Just tried enabling this rule and the error:

```
Cannot read property 'name' of undefined
TypeError: Cannot read property 'name' of undefined
    at Object.getFunctionName (xxxxxxxxxxx/node_modules/eslint-plugin-ember-cleanup/lib/utils/node.js:90:21)
    at EventEmitter.CallExpression (xxxxxxxxxxx/node_modules/eslint-plugin-ember-cleanup/lib/rules/no-empty-declaration.js:27:13)
```

works fine with the rule disabled.

**Comment by onechiporenko** [comment]
Hi, @kellyselden. Thanks for report!
Could you pls provide a checked code that cause this error?

**Comment by kellyselden** [comment]
Sorry I don't have the bandwidth right now to track it down. Just thought I'd post the stack trace for some visibility. Feel free to close it if you wish.

**Comment by kellyselden** [comment]
It's failing for code that looks like `_this.get('callback')(event, data, previewId, index, _this)`

**Comment by onechiporenko** [comment]
I've published new version 1.7.1

**Comment by kellyselden** [comment]
Thank you. It works.
